HIP/hip-412.md

```diff
@@ -310,7 +481,7 @@ An example similar to the above one. Format "none" is used to indicate that this
 }
 ```
 
-#### Example Schema: opensea
+### Example Schema: opensea
```

Comment:
Is HIP412@1.0.0 basically just the opensea standard? Should we delete this section to avoid confusion?